packages/elements/src/components/ui/buttons/PrimaryButton.tsx

```diff
@@ -76,6 +76,7 @@ export const PrimaryButton = forwardRef<HTMLButtonElement, PrimaryButtonProps>(
           leftIcon={leftIcon}
           left={left}
           right={right}
+          iconSize={size}
```

Comment:
Since we removed the `.large.iconButton` class which says the icon size should be large if a large button only has an icon and no label, we should add that logic here instead.

Something along the lines of:

```suggestion
          iconSize={size === "large" ? (hasLabel ? "medium" : "large") : size}
```

Possibly move that logic somewhere else and make it more readable 😄 

Also do the same for `PrimaryButtonLink`.